sdk/core/core-amqp/review/core-amqp.api.md

```diff
@@ -18,6 +17,7 @@ import { MessageProperties } from 'rhea-promise';
 import { Receiver } from 'rhea-promise';
 import { ReceiverOptions } from 'rhea-promise';
 import { ReqResLink } from 'rhea-promise';
+import { Message as RheaMessage } from 'rhea-promise';
```

Comment:
We can keep this as simply `Message`. If we want to make it `RheaMessage`, then we should do the same for `MessageHeader` and `MessageProperties`  in line 15 and 16 above